src/Parse.y

```diff
@@ -2,16 +2,20 @@
 
   this is a happy grammar :)
 
+  14 shift/reduce conflicts in a single state are expected, due
+  to the ambiguity of if/then/else expressions.
```

Comment:
can we get rid of them by saying they are left associative?

Reply:
no, since if/then/else isn't a binary operation. we could probably rewrite the grammar to be unambiguous but that would needlessly complicate things.